src/com/palmergames/bukkit/towny/command/NationCommand.java

```diff
@@ -1250,7 +1250,7 @@ public void sendList(CommandSender sender, List<Nation> nations, int page, int t
 		if ((page * 10) > nations.size()) {
 			iMax = nations.size();
 		}
-		List<String> nationsordered = new ArrayList();
+		List<String> nationsordered = new ArrayList<>();

```

Comment:
For micro-performance, you could set the initial capacity to `iMax - (10*page) + 10`

Reply:
Can't we set the initialCapacity to 10?